td-streamer-global.inc

```diff
@@ -1295,4 +1304,47 @@ stock TDS_TextDrawGetPreviewVehCol(Text:text, &color1, &color2) {
 #else
     #define _ALS_TextDrawGetPreviewVehCol
 #endif
-#define TextDrawGetPreviewVehCol TDS_TextDrawGetPreviewVehCol
\ No newline at end of file
+#define TextDrawGetPreviewVehCol TDS_TextDrawGetPreviewVehCol
+
+
+stock TDS_TextDrawSetExtraID(Text:text, extra1, extra2 = 0) {
+    new id = _:text, data[E_TD_STREAMER_GLOBAL_DATA];
+
+    if (!_GetTextDrawData(id, data)) {
+        return 0;
+    }
+
+    data[E_TD_EXTRA_1] = extra1;
+    data[E_TD_EXTRA_2] = extra2;
+
+    _SetTextDrawData(id, data);
+    return 1;
+}
+
+#if defined _ALS_TextDrawSetExtraID
+    #error TextDraw streamer must be the first library to hook any textdraw functions!
+#else
+    #define _ALS_TextDrawSetExtraID
+#endif
+#define TextDrawSetExtraID TDS_TextDrawSetExtraID
```

Comment:
The ALS hooking patterns aren't needed here, just name the functions with their proper names.